community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -454,7 +454,7 @@ def create_reservation(lkp: util.Lookup, reservation_name: str, node: str, start
     util.run(f"{lkp.scontrol} create reservation user=slurm starttime={formatted_start_time} duration=180 nodes={node} reservationname={reservation_name}")
 
 
-def get_slurm_reservation_maintenance(lkp: util.Lookup) -> Dict[str, datetime]:
+def get_slurm_reservation_maintenance(lkp: util.Lookup, valid_nodeset: Set) -> Dict[str, datetime]:
```

Comment:
We shouldn't filter by nodeset here. Consider case of nodeset getting its `create_maintenance_reservation` turned off, we will need to remove created reservations.

Reply:
That's a good point. reconfiguration will break this. Updated the code to only conside this flag when we create reservation. lmkwyt